pipelines/managed/push-artifacts-to-cdn/README.md

```diff
@@ -20,3 +20,9 @@ It uses InternalRequests so that it can be run on both public and private cluste
 | verify_ec_task_bundle           | The location of the bundle containing the verify-enterprise-contract task                              | No       | -               |
 | taskGitUrl                      | The url to the git repo where the release-service-catalog tasks to be used are stored                  | Yes | https://github.com/konflux-ci/release-service-catalog.git |
 | taskGitRevision                 | The revision in the taskGitUrl repo to be used                                                         | No       | -               |
+
+## Changes in 0.2.0
+* Update all tasks that now support trusted artifacts to specify the stepAction* parameters
+
+## Changes in 0.1.0
```

Comment:
Why are there two different versions in the changelog if the task goes from 0.0.1 to 0.2.0? I've seen it in others.

Reply:
Shouldn't then be removed or put together? It's ok to me if you want to leave it like this.

Reply:
yes. in my latest rebase, I cleaned all the pipelines up to put them together